index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Odin_Recipes</title>
+</head>
+<body>
+    <h1>Odin Recipes</h1>
+    <ul>
+        <li><a href="recipes/lasagna.html">lasagna_Recipe</a></li>
+        <li><a href="recipes/Pizza.html">Pizza_Recipe</a></li>
+        <li><a href="recipes/Momo.html">Momo_Recipe</a></li>
```

Comment:
we should not snake_case string that users would be seeing. These should be in Title Case - `Lasagna Recipe`, `Pizza Recipe` etc.